container/setup.py

```diff
@@ -0,0 +1,48 @@
+"""A setup module for the GAPIC Google Container Engine API library.
```

Comment:
This needs a license header.

Reply:
Not obviously shown for whatever reason, but I added one.